src/WSSlots.php

```diff
@@ -64,9 +96,14 @@ final public static function editSlots(
 		User $user,
 		WikiPage $wikiPage,
 		array $slotUpdates,
-		string $summary,
+		string $summary = '',
 		bool $append = false,
-		string $watchlist = ""
+		bool $prepend = false,
+		string $watchlist = "",
```

Comment:
Note: This is a backwards-compatibility break: prepend arg is where watchlist used to be.